src/main/java/HibUtil/HibernateUtil.java

```diff
@@ -0,0 +1,27 @@
+package HibUtil;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+    private static final SessionFactory sessionFactory;
+//    private static final ServiceRegistry serviceRegistry;
```

Comment:
Czym jest serviceRegistry i dlaczego jest zakomentowane? Jak nie będziemy używać to do wywalenia wszystkie komentarze z tego pliku